nekoyume/Assets/_Scripts/UI/Widget/Popup/ArenaTicketPopup.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Nekoyume.Game;
+using Nekoyume.State;
+using Nekoyume.UI.Module;
+using TMPro;
+using UniRx;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI
+{
+    public class ArenaTicketPopup : PopupWidget
+    {
+        [SerializeField]
+        private SweepSlider ticketSlider = null;
+
+        [SerializeField]
+        private TextMeshProUGUI haveTicketText = null;
+
+        [SerializeField]
+        private TextMeshProUGUI ticketCountToUseText = null;
+
+        [SerializeField]
+        private ConditionalButton startButton = null;
```

Comment:
Is the design of the button is pretty different between enabled state and disabled state?

Reply:
If the button is disabled, it is dimmed.
![image](https://user-images.githubusercontent.com/63496908/213064708-c331b32b-565d-427c-838f-8c173faed571.png)